features/support/parameter_types.rb

```diff
@@ -474,3 +474,14 @@ def find_metric(metrics, name)
   regexp: /(.*)/,
   transformer: ->(selector) { selector_for(selector) }
 )
+
+ParameterType(
+  name: 'read_only_field',
```

Comment:
I wonder if this can be improved slightly :thinking: 
```suggestion
  name: 'read_only_status',
```